setup-container.sh

```diff
@@ -0,0 +1,96 @@
+#! /bin/bash
+
+function show_help_and_exit() {
+    echo "Usage $0 [options]"
+    echo "Options with (*) are required"
+    echo "-h -?                 : get this help"
+    echo "-n <container name>   : (*) set the name of the Docker container"
+    echo "-i <image ID>         : (*) specify Docker image to use"
```

Comment:
Can you put in an internal PR to also have a default image if user doesn't specify any since many users would just want to have a docker setup for running tests and would want to avoid the hassle of finding the right image to use?

Reply:
I think most people are using the public mgmt docker anyways so you could probably get away with pulling it from here: https://sonic-jenkins.westus2.cloudapp.azure.com/job/bldenv/job/docker-sonic-mgmt/lastSuccessfulBuild/artifact/sonic-buildimage/target/

Reply:
I'm not 100% sure how to approach this. As far as I can tell, there's no way to grab the image ID of the latest image from the public Jenkins, which means there's no way to check if that image already exists on the server before downloading it. If the image does exist already, Docker will automatically remove the name/tag from an existing image named "docker-sonic-mgmt" if it exists, which would result in possibly overwriting internally built images. This could also potentially result in many untagged images taking up disk space.

Reply:
That's a good point! We can revisit that later, I think the script is still helpful to have even without the auto-download step. :)

Reply:
@lguohan Having a docker registry where we can store the latest public image would make this feature pretty simple to implement (as suggested by @daall). Would it be possible to set one up and upload public sonic-mgmt images after they're built?

Reply:
yes, i believe we are already doing that.